### PR TITLE
Add initial implementation.

### PR DESCRIPTION
**Pull request by hgaiser**
Should we also add corner cases? For example: https://github.com/mrdoob/three.js/blob/dev/src/math/Euler.js#L118 that check is currently not covered by this implementation.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+///
+/// # Arguments
+///
+/// * `rx` - The X component of the axis angle vector.
+/// * `ry` - The Y component of the axis angle vector.
+/// * `rz` - The Z component of the axis angle vector.
+/// * `angle` - The rotation around the axis angle vector to apply in radians.
+///
+/// Known robots that use this format:
+/// - Mitsubishi Industrial Robot CR750/CR751 Series
```
Comment (vcarpani) [inline src/main.rs:114]:
```suggestion
/// Known robots that use this format:
/// - Mitsubishi Industrial Robot CR750/CR751 Series
/// - Kawasaki, after conversion to degrees
/// - Kuka, after conversion to degrees
```
Source: https://doc.rc-visard.com/v21.07/en/pose_format_kuka.html#:~:text=KUKA%20robots%20use%20the%20so,)rx(C).

Comment (hgaiser) [reply src/main.rs:114]: Reworded it a bit, but done :+1:

**Review by vcarpani — APPROVED** [review]

**Comment by de-vri-es** [comment]
Maybe we should add functions per robot that also do conversion to degrees where applicable? But I would suggest that we call the radians based version rather than duplicating it completely.

**Comment by hgaiser** [comment]
> Maybe we should add functions per robot that also do conversion to degrees where applicable? But I would suggest that we call the radians based version rather than duplicating it completely.

Hmm I would prefer to put that responsibility in the hands of the integrator.

**Comment by de-vri-es** [comment]
> Hmm I would prefer to put that responsibility in the hands of the integrator.

I think it would be a nice way to document the conventions for each robot, rather than having the integrator search through comments for notes on angle units. It could save us some questions for the support team.

But either way: could you disable the clippy lint about unneeded return statements for this code?

**Comment by hgaiser** [comment]
> I think it would be a nice way to document the conventions for each robot, rather than having the integrator search through comments for notes on angle units. It could save us some questions for the support team.

How did you have this in mind? A function like `fn convert_axis_angle_to_euler_yaskawa(...)`?

**Comment by de-vri-es** [comment]
> How did you have this in mind? A function like `fn convert_axis_angle_to_euler_yaskawa(...)`?

Yeah, exactly that.

**Review by vcarpani — APPROVED** [review]

**Comment by de-vri-es** [comment]
So what about these conditionals:

https://github.com/mrdoob/three.js/blob/c3978640fef2e2722f1fa30fe2369eb161a4dfb6/src/math/Euler.js#L120-L130

Are we sure they're just an optimization and not required?

Could we add more unit tests that use for example nalgebra to verify the implementation? It would make me sleep better at night.

**Comment by de-vri-es** [comment]
I have some proposed changes in #2. That PR will merge into this branch if accepted.

Based on those tests it does look like the conditional is not required.

**Review by de-vri-es — APPROVED** [review]

**Pull request merged**